### ISSUE TITLE
Compilation with `USE_PERF_JITEVENTS=1` fails

### ISSUE THREAD

**Issue by athre0z** [JuliaLang/julia#33417]
**Platform:** Ubuntu 18.04.3 LTS
**Tested Julia versions:**  `v1.2.0`, `v1.3.0-rc2` as well as latest master (as of writing, d0365753fb57f57bcea097c05de8c07aef706561)

Adding `USE_PERF_JITEVENTS=1` to  Make.user results in a build error while linking against some LLVM symbol:

```
parallels@ubuntu /media/psf/Home/Development/julia (git)-[master] % make         
    LINK usr/lib/libjulia.so.1.4
./codegen.o: In function `jl_init_llvm':
/media/psf/Home/Development/julia/src/codegen.cpp:7761: undefined reference to `llvm::JITEventListener::createPerfJITEventListener()'
collect2: error: ld returned 1 exit status
Makefile:311: recipe for target '/media/psf/Home/Development/julia/usr/lib/libjulia.so.1.4' failed
make[1]: *** [/media/psf/Home/Development/julia/usr/lib/libjulia.so.1.4] Error 1
Makefile:75: recipe for target 'julia-src-release' failed
make: *** [julia-src-release] Error 2
```

Without this option, the build works fine.

**Comment by vchuravy** [comment]
What is you Make.user? You might have to set `USE_BINARYBUILDER_LLVM=0`

**Comment by athre0z** [comment]
My Make.user previously only consisted of the one `USE_PERF_JITEVENTS=1` entry. Adding `USE_BINARYBUILDER_LLVM=0` indeed resolved the issue, thank you!

It might be worth adding this to the documentation that talks about the perf event support here:
https://docs.julialang.org/en/v1/manual/profile/#External-Profiling-1

**Comment by vchuravy** [comment]
Yeah would be good to add an note there and maybe even default to ship it with HITEVENTS for VTune and perf enabled.

**Comment by athre0z** [comment]
If that doesn't cause any noticeable slowdowns when disabled at run-time, I feel like that would be best indeed!
